packages/common/src/types/certificate.ts

```diff
@@ -98,6 +98,7 @@ export enum ConfigTypes {
   UE_VIEW_LIST = 'UE_VIEW_LIST',
   UE_VIEW_TABLE = 'UE_VIEW_TABLE',
   UE_YEAR = 'UE_YEAR',
+  UE_INTEGER = 'UE_INTEGER',
 }
 
```

Comment:
There are changes in the master branch that requires some modifications to the type declarations fot configs, so you should merge the master branch into this now and implement these